docs/user/ppl/functions/datetime.rst

```diff
@@ -813,6 +813,78 @@ Example::
     +-----------------------------------+
 
 
+EXTRACT
+-------
+
+Description
+>>>>>>>>>>>
+
+Usage: extract(part FROM date) returns a LONG with digits in order according to the given 'part' arguments.
+The specific format of the returned long is determined by the table below.
+
+Argument type: PART
```

Comment:
Missing new line I think
<img width="644" alt="Screenshot 2023-07-06 at 5 13 41 PM" src="https://github.com/Bit-Quill/opensearch-project-sql/assets/29149268/3eead716-8db9-4668-b5f3-c03eafdcb395">

Reply:
Good catch, I updated the documentation in https://github.com/Bit-Quill/opensearch-project-sql/pull/283/commits/872ece9c46db0eab82c2f34ed3cf7d973947c02a